lib/send_messages/send_messages.js

```diff
@@ -1,4 +1,4 @@
-exports.sendMessage = function(userId, password, content, numbers, callback) {
+exports.sendMessage = function(userId, password, content, numbers, deliveryReport, scheduledTime, messageId, callback) {

```

Comment:
You mean if somebody already wrote some code using this lib it will break theirs code? Ok, I will put these new params in the end.